packages/electrode-archetype-react-app-dev/config/webpack/partial/dev.js

```diff
@@ -8,11 +8,24 @@ const webpackDevReporter = require("../util/webpack-dev-reporter");
 const devProtocol = process.env.WEBPACK_DEV_HTTPS ? "https://" : "http://";
 
 module.exports = function () {
+  const devServerConfig = {
+    reporter: webpackDevReporter,
+    https: Boolean(process.env.WEBPACK_DEV_HTTPS)
+  };
+  if (process.env.WEBPACK_HOST) {
+    devServerConfig.host = process.env.WEBPACK_HOST;
```

Comment:
these are picked up in `config/archetype.js`, available under `webpack.devHostname`.